pkg/plugins/resources/toolversions/main.go

```diff
@@ -0,0 +1,64 @@
+package toolversions
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/updatecli/updatecli/pkg/core/text"
+	"github.com/updatecli/updatecli/pkg/plugins/utils/toolversions"
+)
+
+var (
+	// ErrDaselFailedParsingToolsVersionsByteFormat is returned if dasel couldn't parse the byteData
+	ErrDaselFailedParsingToolsVersionsByteFormat error = errors.New("fail to parse .tool-versions data")
+)
+
+// ToolVersions stores configuration about the file and the key value which needs to be updated.
+type ToolVersions struct {
+	spec     Spec
+	contents []toolversions.FileContent
```

Comment:
Do I need [`505b162` (#2265)](https://github.com/updatecli/updatecli/pull/2265/commits/505b162e626871f49120ba94b419b97140c49f85)?

Reply:
Do you mean `foundVersion version.Version`?
I don't think so, as far as I understand this plugin, you don't need to filter version based on rules like semantic versioning ,etc.

Reply:
> Do you mean foundVersion version.Version?

Yes. Thanks for the clarification :)